src/encoded/schemas/gene.json

```diff
@@ -0,0 +1,107 @@
+    "identifyingProperties": ["uuid", "geneid"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/standard_status" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "geneid": {
+            "title": "NCBI Entrez GeneID",
+            "description": "NCBI Entrez GeneID of official nomenclature approved gene.",
+            "comment": "An ENCODE gene (object) needs to have an Entrez GeneID and needs to be approved by (i.e. have an ID and gene symbol from) offical nomenclature such as HGNC, MGI, FlyBase, WormBase.",
+            "type": "string",
+            "permission": "import_items",
+            "uniqueKey": true
+        },
+        "symbol": {
+            "title": "Gene symbol",
+            "description": "Gene symbol approved by the official nomenclature.",
+            "type": "string",
+            "format": "gene_name",
+            "permission": "import_items"
+        },
+        "synonyms": {
+            "title": "Synonyms",
+            "description": "Alternative symbols that have been used to refer to the gene.",
+            "comment": "The list of synonyms is maintained using information from NCBI Entrez. Synonyms not included in NCBI gene_info can be added but are not actively maintained.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "Synonym",
+                "description": "An alternative symbol/name referring to the gene.",
+                "type":  "string",
+                "format": "gene_name"
+            },
+            "permission": "import_items"
+        },
+        "dbxrefs": {
+            "title": "External identifiers",
+            "description": "Unique identifiers from external resources.",
+            "comment": "This list should have at least an ID from the official nomenclature. The list of other dbxrefs is maintained using information from NCBI Entrez. Identifiers not included in NCBI gene_info can be added but are not actively maintained.",
+	          "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource (e.g. HGNC, MGI, FLYBASE, WormBase, Ensembl, MIM, RefSeq, UniProtKB, Vega, miRBase).",
+                "comment": "Submit as database_name:id. See changelog for potential databases.",
+                "type":  "string",
+                "pattern": "^((HGNC:HGNC:[0-9a-zA-Z/\\-]+)|(MGI:MGI:\\d+)|(FLYBASE:[0-9a-zA-Z\\-]+)|(WormBase:[0-9a-zA-Z\\.]+)|(Ensembl:ENS[A-Z]*G\\d+)|(MIM:\\d+)|(RefSeq:[A-Z]{2}_\\d+)|(UniProtKB:[0-9A-Z]+)|(Vega:OTT[A-Z]+G\\d+)|(miRBase:MI\\d+))$"
```

Comment:
The pattern for FlyBase and Ensembl should be "FlyBase" and "ENSEMBL" to match what's in the namespace schema.